.vib/zookeeper/goss/zookeeper.yaml

```
@@ -0,0 +1,5 @@
+{{ $serverBin := "/opt/bitnami/zookeeper/bin/zkServer.sh" }}
+command:
+  check-custom-daemon-config:
+    exec: cat {{ $serverBin }} | grep -q "^_ZOO_DAEMON" && cat {{ $serverBin }} | grep -q '_ZOO_DAEMON_OUT=$ZOO_LOG_DIR/zookeeper.out'
```

Why is the first part needed?

```suggestion
    exec: cat {{ $serverBin }} | grep -q '_ZOO_DAEMON_OUT=$ZOO_LOG_DIR/zookeeper.out'
```

---

This is a mistake, I misinterpreted our internal compilation logic. Fixed now!